src/resources/js/v2/rsvp-block.js

```diff
@@ -138,6 +135,35 @@ tribe.tickets.rsvp.block = {};
 		} );
 	};
 
+	/**
+	 * Handle the RSVP toggle for listing in public attendee list.
+	 *
+	 * @since TBD
+	 *
+	 * @param {event} e submission event
+	 */
+	obj.handleDisplayToggle = function( e ) {
+		e.preventDefault();
+
+		const $input = $( e.target );
+		const rsvpId = $input.data( 'rsvp-id' );
+		const checked = $input.prop( 'checked' );
+		const attendeeIds = $input.data( 'attendee-ids' );
+		const nonce = $input.data( 'opt-in-nonce' );
+		const $container = e.data.container;
+
+		var data = {
+			action: 'tribe_tickets_rsvp_handle',
```

Comment:
Wonder if we could store this action name in an attribute of this object, similar to how we approach our PHP.

Reply:
We already have it this way in all our different JS files across projects. I feel we should keep consistency with how we have the JS files and not PHP.

Reply:
I'm also okay with leaving this as it is for now, but if there's anything we need to do to improve this later across our JS that'd be a cool ticket to work on